conference/react-js/src/containers/SignUpForm/SignUpForm.jsx

```diff
@@ -15,27 +15,29 @@ class SignUp extends Component {
       mail: '',
       password: '',
       confirm: '',
+      passLen: false,
```

Comment:
not critical, but for future, use full self-descriptive names.

Reply:
ok, that's mine, I'll change it)